src/linux/elf.rs

```diff
@@ -178,21 +163,6 @@ impl InlineeManager {
             info
         }
     }
-
-    /// Set the location where the inlinees are inlined in the inliner
-    /// We use this function when some addresses in an inlinee are before the function start
-    fn set_line_info(&mut self, line: &LineInfo, compilation_dir: &[u8], source: &mut SourceFiles) {
-        if let Some(inlinee_pos) = self.addresses.get(&line.address) {
-            let inlinee = &mut self.inlinees[*inlinee_pos];
-            if inlinee.start == line.address {
-                let file_id = source.get_id(compilation_dir, &line.file);
-                inlinee.location = ElfLineInfo {
-                    file_id,
-                    line: line.line as u32,
-                };
-            }
-        }
```

Comment:
This and the other changes to the inlined functions seem unrelated to the multi-file support. Is this the functionality that was recently merged into symbolic? If it is it would be best to split it out into another patch for clarity.

Reply:
I moved that stuff in a separate commit.